src/test/java/teammates/test/cases/webapi/GetSessionResultsActionTest.java

```diff
@@ -0,0 +1,130 @@
+package teammates.test.cases.webapi;
```

Comment:
Is this test related to your PR?

Reply:
Not exactly. But since I'm writing `GetSessionResultsAsCsvActionTest`, I thought it would be ok just to add `GetSessionResultsActionTest` at the same time because the test for `GetSessionResultsAction` is currently missing. There shouldn't be any breaking changes to existing code.